examples/material-ui-cra/package.json

```diff
@@ -9,12 +9,12 @@
     "eject": "react-scripts eject"
   },
   "dependencies": {
-    "@emotion/react": "latest",
-    "@emotion/styled": "latest",
-    "@mui/material": "latest",
-    "react": "latest",
-    "react-dom": "latest",
-    "react-scripts": "latest"
+    "@emotion/react": "^11.0.0",
+    "@emotion/styled": "^11.0.0",
+    "@mui/material": "^5.0.0",
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0",
+    "react-scripts": "^5.0.0"
```

Comment:
I think we can use latest here, as we plan to bring the React 19 support to v5 too. @DiegoAndai what are your thoughts?

Reply:
Ah yeah, I didn't realize we were planning on bringing React 19 to v5. I assumed that because we released v6 before React 19 is stable, we wouldn't need to spend time on this. At least, I believe we fulfilled all the conditions to have React 19 only in v6 https://github.com/mui/material-ui/issues/42381#issuecomment-2150903771.

Now, no real preferences on my end. I think https://tools-public.mui.com/prod/pages/npmVersion?package=@mui/material could tell us what's the best move. React 19 could help us consolidate the community on a single major.

Reply:
I'm reverting the "react" changes for now, it's fewer changes, it's never bad to scope down PRs 😄